apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/DataLayoutStrategyGeneratorSparkApp.java

```diff
@@ -41,15 +46,39 @@ protected void runInner(Operations ops) throws Exception {
         strategies.stream().map(Object::toString).collect(Collectors.joining(", ")));
     StrategiesDao dao = StrategiesDaoTableProps.builder().spark(spark).build();
     dao.save(fqtn, strategies);
+    if (outputFqtn != null && !strategies.isEmpty()) {
+      List<String> rows = new ArrayList<>();
+      for (DataLayoutStrategy strategy : strategies) {
+        rows.add(
+            String.format(
+                "('%s', current_timestamp(), %f, %f, %f, %f, '%s')",
+                fqtn,
+                strategy.getCost(),
+                strategy.getGain(),
+                strategy.getEntropy(),
+                strategy.getScore(),
+                SerDeUtil.toJsonString(strategy)));
+      }
+      String strategiesInsertStmt =
+          String.format("INSERT INTO %s VALUES %s", outputFqtn, String.join(", ", rows));
+      log.info("Running {}", strategiesInsertStmt);
```

Comment:
Couple of comments:

1. Can we create table with schema if it does not exists? Here in the code. For completeness.
2. Should do an UPSERT on table name and/or set Retention (as part of table creation) on the timestamp column? 

For table scope we will insert one row at a time right?

Reply:
1) ok; 2) can set retention, not sure about upsert, we can have all the records, multiple per day if needed, and filter out at analysis step.
We'll change this when we add scope.

Reply:
actually, it's better to set retention externally, it's a separate operation

Reply:
Ok, but we need the table to be partitioned atleast. Right?

Reply:
Yes, partitioned is created